src/tests/Microsoft.PowerFx.Connectors.Tests/InternalTesting.cs

```diff
@@ -1323,6 +1362,108 @@ public YamlMapping()
 
     public static class Exts
     {
+        public static string GetString(this OpenApiSchema schema)
```

Comment:
>GetString [](http://example.com/codeflow?start=29&length=9)

This allows us to store key elements of the schema in our SQL database as it is presented to us